celery/result.py

```diff
@@ -130,6 +130,8 @@ def as_tuple(self):
     def forget(self):
         """Forget about (and possibly remove the result of) this task."""
```

Comment:
Please adjust the docstring to specify that parent tasks will also be forgotten.

Reply:
OK